### ISSUE TITLE
Bad link to the privacy policy page in the footer

### ISSUE THREAD

**Issue by mscherer** [oVirt/ovirt-site#3]
So on the new website, it seems that the link to the privacy policy is missing and give a 404.

**Comment by garrett** [comment]
Link to https://www.redhat.com/en/about/privacy-policy ?

**Comment by garrett** [comment]
(I think the conversion of http://www.ovirt.org/oVirt_Wiki:Privacy_policy was skipped as it was a special page... and I'm not sure how much it applies now that there isn't a MediaWiki authentication system as a part of the site.)

**Comment by mscherer** [comment]
I am not sure if reusing the privacy policy of RH is a good way to show "that's a independent community", given it say that RH only do things with the data. There is also several breach of the privacy policy in practice ( like the email sent on infra ml with part of the log, for example ).

**Comment by garrett** [comment]
This is what's currently on oVirt.org:
> By joining the oVirt community, you will be giving us some personal information, including your name and an email address. Our use of such information is governed by a privacy policy established by Red Hat.

(from http://www.ovirt.org/oVirt_Wiki:Privacy_policy)

---

It basically links to the Red Hat privacy policy right now, and has something that is applicable to the oVirt wiki, but not to the new site. So... what should be done for the new site?

**Comment by thatdocslady** [comment]
@mscherer - the link from the Home page should go to the page with the text that @garrett pasted above.

As for what to write, unless anyone has any brilliant and easy-to-implement ideas, we can continue with the existing RH policy text. RDO also uses this method: https://www.rdoproject.org/legal/

**Comment by mscherer** [comment]
So coming back on that issue, the RH policy say "This Privacy Statement may not apply to open source project websites sponsored by Red Hat.". This is ambigous, because it can be "may not" as "is not permitted to" or "is not automatically supposed to". 

The RH policy also say we do not give the information except:
- business partner, who signed a confidentiality agreement
- merger/acquisition/etc
- law enforcement requests.

having community admins enter none of those category, so we are violating the policy quite often.

I also find the part about remarketing to be unsettling, and I am quite sure we do not want to do that on our community websites.

The policy also explain how to change the personal information, which is a likely incorrect way (since it go to redhat web team, who likely have no access on ovirt.org infra).

There is also a bit about safe harbor which is bogus since this framework was dissolved, but RH is already working on that. 

In short, nope, just a link is bad. I can't in good faith say we take privacy seriously given how lightly we treat that. I understand that a fix would take time, but so far, we aren't even agreeing there is a problem despites them being easy to spot by just reading the policy.

**Comment by thatdocslady** [comment]
@mscherer @garrett can we separate the broken link issue from the privacy policy issue? if the link is fixed, let's close this issue and open another one if further discussion about privacy policy is required

**Comment by thatdocslady** [comment]
It looks like @mscherer opened another issue for the privacy policy: https://github.com/oVirt/ovirt-site/issues/55

@garrett can you please help fixing the broken link and then close this issue? thanks!

**Comment by sandrobonazzola** [comment]
@mscherer @garrett @thatdocslady Looks like privacy policy link has been fixed. Closing for now.
